.github/workflows/ci.yml

```diff
@@ -553,8 +553,10 @@ jobs:
           hello
           nix store gc
           nix run nixpkgs#hello
+      # NOTE(cole-h): GHA pushed a weird image that breaks this test for whatever reason, so ignore
+      # the failure for now
```

Comment:
This seems related: https://github.com/actions/runner-images/issues/10624

(Not putting this in the comment so we don't need another run of slow x86_64-darwin CI)